mkosi.md

```diff
@@ -812,6 +812,12 @@ details see the table below.
   with this  option and running the image using `mkosi boot` or `mkosi qemu`,
   the `mkosi ssh` command can be used to connect to the container/VM via SSH.
 
+`--ssh-private-key=`
```

Comment:
I think this could safely be called `--ssh-key=` now.

Reply:
Quite indeed!